.travis.yml

```diff
@@ -0,0 +1,62 @@
+---
+language: ruby
+
+rvm:
+  - 1.8.7
```

Comment:
I'd prefer not to test 1.8.7 because its really old and not even maintained.

This module is for preferably going to be for Puppet V4 only though it can be configured to work for v3 as well.